packages/reactDom/src/ReactServerDOM.ml

```diff
@@ -54,6 +54,7 @@ module Model = struct
   let lazy_value id = Printf.sprintf "$L%x" id
   let promise_value id = Printf.sprintf "$@%x" id
   let ref_value id = Printf.sprintf "$%x" id
+  let error_value id = Printf.sprintf "$Z%x" id
```

Comment:
in the RSC model I see the E being used as error, something among this format:

```
3:E{"digest":"","name":"Error","message":"lol","stack":[["AlwaysError","/Users/davesnx/Code/github/ml-in-barcelona/server-reason-react/arch/server/render-rsc-to-stream.js",35,6]],"env":"Server"}
```

where does Z come from?